plugins/services/src/js/constants/ContainerConstants.js

```diff
@@ -5,6 +5,6 @@ module.exports = {
   },
   labelMap: {
     DOCKER: "Docker Engine",
-    MESOS: "Mesos Runtime"
+    MESOS: "Universal Container Runtime (UCR)"
```

Comment:
Since we use this in so many places. Could we probably use a template literal when we use this?
E.x. Docker Runtime only supports the default size for implicit volumes, please select ${Container.MESOS} if you want to modify the size.

I can see @weblancaster not agreeing with this a mile away :)

Reply:
I agree that refactoring to use template strings is a more maintainable approach. When I made this simple string change, it broke tests across many many files. Our cypress tests are written in a very "always repeat yourself" way. But I think in this case, since this string constant already exists, we should use the constant in a template string.